tests/end2end/features/tabs.feature

```diff
@@ -660,6 +660,59 @@ Feature: Tab management
         And I run :tab-detach
         Then the error "Cannot detach one tab." should be shown
 
+    # :tab-attach
+
+    Scenario: Attach tab to first-opened window
+        When I open data/numbers/1.txt
+        And I open data/numbers/2.txt in a new window
+        And I open data/numbers/3.txt in a new window
+        And I run :buffer "3.txt"
+        And I run :tab-attach
+        Then the session should look like:
+            windows:
+            - tabs:
+              - history:
+                - url: about:blank
+                - url: http://localhost:*/data/numbers/1.txt
+              - history:
+                - url: http://localhost:*/data/numbers/3.txt
+            - tabs:
+              - history:
+                - url: http://localhost:*/data/numbers/2.txt
+
+    Scenario: Attach tab to arbitrary window
+        Given I have a fresh instance
+        When I open data/numbers/1.txt
+        And I open data/numbers/2.txt in a new window
+        And I open data/numbers/3.txt in a new window
+        And I run :buffer "3.txt"
+        And I open data/numbers/4.txt in a new tab
+        And I run :tab-attach 1
+        Then the session should look like:
+            windows:
+            - tabs:
+              - history:
+                - url: about:blank
+                - url: http://localhost:*/data/numbers/1.txt
+            - tabs:
+              - history:
+                - url: http://localhost:*/data/numbers/2.txt
+              - history:
+                - url: http://localhost:*/data/numbers/4.txt
+            - tabs:
+              - history:
+                - url: http://localhost:*/data/numbers/3.txt
```

Comment:
Looks like those two tests fail on CI, and there's probably nothing that can be done about it until #1844 is working and in. I suggest adding `@xfail_norun` above `Scenario:`, and a comment like:

```python
# Needs qutewm to run properly
# See https://github.com/qutebrowser/qutebrowser/pull/1844
```